source/install/cloud-changelog.md

```diff
@@ -3,12 +3,58 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-08-11](#release-2021-08-11)
 - [Release 2021-07-29](#release-2021-07-29)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
 - [Release 2021-06-16](#release-2021-06-16)
 - [Release 2021-06-02](#release-2021-06-02)
-- [Release 2021-05-21](#release-2021-05-21)
+
+## Release 2021-08-11
+
+### Highlights
+
+#### Added a global product launcher
+ - 
```

Comment:
Left this blank for now as this setting may be left as disabled.

Reply:
@amyblais we should be keeping this feature flag disabled until Aug 16th I imagine